notification-server/src/test/java/uk/ac/bbsrc/tgac/miso/notification/service/IlluminaTransformerTest.java

```diff
@@ -228,7 +230,7 @@ public void testDateCompletedViaCycleTimes() throws JSONException {
 
     assertTrue(run.has(IlluminaTransformer.JSON_COMPLETE_DATE));
     String date = run.getString(IlluminaTransformer.JSON_COMPLETE_DATE);
-    assertTrue(date != null && !date.equals(""));
+    assertTrue(!isStringEmptyOrNull(date));
     assertTrue(!date.equals("null"));
```

Comment:
Out of scope I guess for this request but these tests could check for a valid date.